docs/sources/source/nlp/transformer_language_model.rst

```diff
@@ -15,7 +15,7 @@ Download Corpus
 
 For demonstration purposes, we will be using the very small WikiText-2 dataset :cite:`nlp-lm-merity2016pointer`.
 
-To download the dataset, run the script ``examples/nlp/scripts/get_wt2.sh``. After downloading and unzipping, the folder should include 3 files that look like this:
+To download the dataset, run the script ``examples/nlp/scripts/get_wkt2.sh``. After downloading and unzipping, the folder should include 3 files that look like this:
```

Comment:
the script name is get_wt2.sh

Reply:
It is get_wtk2.sh in my version. Checked the master too.

Reply:
ok, probably this was renamed sometime.